media/components/transformation_list/transformation_list.js

```diff
@@ -43,8 +43,16 @@ class Transformation extends TransformationListItem {
                         this.sdfg_id + '/' + this.state_id + '/' + id + '/-1'
                     );
             }
+        }
+
+        if (uuids.length)
+            return uuids;
+
+        if (this.label === 'NestSDFG' || this.label === 'GPUTransformSDFG')
```

Comment:
why? can we make this a more general check?

Reply:
We can, this is cleaner :-)